### ISSUE TITLE
Suggestion: config for MM/BWM power conversion ratio

### ISSUE THREAD

**Issue by bloche1871** [DaedalusGame/MysticalMechanics#33]
Right now, BWM **windmills and watermills do not generate enough power to power an ember bore from Embers Rekindled, when converted to Mystical Mechanics power** (or most of the other mechanically-powered machines from Embers I tried).

Similarily, **using a steam engine from Embers Rekindled and converting the power to BWM axle power gives too much power for any BWM device to use without breaking.**

This makes the converter very impractical in practice. Having a config to set the conversion ratio would be great to be able to address issues like this. (I am working on a modpack where I'm hoping to use BWM and MM axle power, and conversion between them, as a central element of early-game progression)

(Another issue which does not concern Mystical Mechanics directly but would be addressed by such a config, is that currently the RF dynamo from Mystical Gears does not receive enough power to produce RF if powered by a BWM mill converted to MM axle power (I was hoping to completely replace IE mills in my modpack with the BWM ones, relying on this conversion to produce RF from them)

**Comment by nephatrine** [comment]
Yes, would definitely love a config option for exactly this.

@bloche14  Though not helpful for powering Embers, if you just want to replace IE mills with BWM ones for RF, you can try BetterWithEngineering - which does have a configurable conversion ratio (by default it also uses a really poor ratio for some reason).

**Comment by DaedalusGame** [comment]
i started this a little bit but the conversion rate is actually non-linear so it gave me the potato brain

**Comment by bloche1871** [comment]
@DaedalusGame , do you know if a custom Modular Machine could convert between the two rotation systems correctly? Or is the issue with non-linear conversion rates also a problem there?

**Comment by nephatrine** [comment]
@bloche14  Doesn't modular machinery/diversity just do whatever you tell it to do? I don't think it automatically converts anything. So you'd need to specify an input of one power and an output of the other and specify that ratio yourself.

**Comment by bloche1871** [comment]
Yes that's what I meant. I was wondering if I would run into the same issue DaedalusGame was having if I tried to do that or if it would work to convert power between the two correctly

**Comment by DaedalusGame** [comment]
you can use mm to convert power yes.
